devicetypes/smartthings/zwave-water-temp-light-sensor.src/zwave-water-temp-light-sensor.groovy

```diff
@@ -0,0 +1,226 @@
+/**
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  Z-Wave Water/Temp/Light Sensor
+ *
+ *  Author: SmartThings
+ *  Date: 2018-08-09
+ */
+
+metadata {
+	definition(name: "Z-Wave Water/Temp/Light Sensor", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "x.com.st.d.sensor.moisture", runLocally: true, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false) {
+		capability "Water Sensor"
+		capability "Sensor"
+		capability "Battery"
+		capability "Health Check"
+		capability "Temperature Measurement"
+		capability "Illuminance Measurement"
+
+		fingerprint mfr: "019A", prod: "0003", model: "000A", deviceJoinName: "Sensative Strips"
```

Comment:
There are three different sensors called "Strips" right? Shouldn't this be "Sensitive Strips Drip"